modell/src/main/kotlin/no/nav/tilbakekreving/hendelse/KravgrunnlagHendelse.kt

```diff
@@ -93,6 +142,13 @@ class KravgrunnlagHendelse(
         MANUELL("Manuell behandling"),
         NYTT("Nytt kravgrunnlag"),
         SPERRET("Kravgrunnlag sperret"),
+        ;
+
+        companion object {
+            fun fraNavn(navn: String): Kravstatuskode {
+                return entries.find { it.navn == navn }!!
```

Comment:
Den kan i teorien være null, greit å kaste feil med forklarende melding i det tilfellet. Alternativet er å gjøre `Kravstatuskode?`
```suggestion
                return entries.find { it.navn == navn }
                    ?: throw IllegalArgumentException("Ingen kravstatuskode med navn=$navn ble funnet")
```